CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Node.js Agent for Google Cloud Trace Changelog
 
+## 2018-05-02, Version 2.8.0 (Beta), @kjin
+
+This version makes some changes to configuration options, as well as the custom span API.
```

Comment:
`changes` sounds like breaking. It would be better to say 'adds new configuration options'.